.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build_windows:
+    name: 🪟 Build Windows App
+    runs-on: windows-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+      - run: flutter pub get
+      - run: flutter build windows --release
+      - name: Upload Windows App to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: build\windows\x64\runner\Release\quran_publisher_desktop.exe
```

Comment:
Flutter Windows app cannot run with only the .exe, it needs .dll files as well. 

![image](https://github.com/user-attachments/assets/dded2d0d-5a61-40b5-ab8f-197c74c73ff9)

Read detail here: https://docs.flutter.dev/platform-integration/windows/building#building-your-own-zip-file-for-windows

Reply:
will check into it